apps/frontend/src/components/global/UploadNexus.vue

```diff
@@ -105,7 +106,7 @@ export default class UploadNexus extends mixins(ServerMixin) {
   // Handles change in tab
   selected_tab(new_tab: string) {
     this.active_tab = new_tab;
-    this.$toasted.clear();
+    SnackbarModule.SET_VISIBILITY(false);
```

Comment:
There should probably be a `visible()` Action on the Vuex store, directly calling Mutations here seems like potentially a bad idea, not sure if I know all of the implications but the general idea is:

> Actions are similar to mutations, the differences being that:
> 
>     Instead of mutating the state, actions commit mutations.
>     Actions can contain arbitrary asynchronous operations.